engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRProvider.java

```diff
@@ -89,10 +91,18 @@ public static OpenVRProvider getInstance() {
         return instance;
     }
 
+    /**
+     * Get the state of the VR system. This contains the poses of the eyes, controllers, etc...
+     * @return the VR state.
+     */
     public OpenVRState getState() {
         return vrState;
     }
 
+    /**
+     * Initialize the VR system. Note that calling this method will cause OpenVR to launch.
+     * @return true if successful.
+     */
```

Comment:
I'd add a few words about the circumstances in which this method will return false.